api/PushRegistrationManager.json

```diff
@@ -0,0 +1,244 @@
+{
+  "api": {
+    "PushMessageData": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PushRegistrationManager",
+        "support": {
+          "webview_android": {
+            "version_added": null
```

Comment:
It looks like this interface, which is deprecated from the spec, has been removed completely from our code base. Please change all Chrome and Opera values to `false`. (I don't have version removed number.)